doc/developers/advanced_installation.rst

```diff
@@ -120,25 +120,80 @@ Mac OSX
 -------
 
 The default C compiler, Apple-clang, on Mac OSX does not directly support
-OpenMP. The first solution to build scikit-learn is to install another C
-compiler such as gcc or llvm-clang. Another solution is to enable OpenMP
-support on the default Apple-clang. In the following we present how to
-configure this second option.
+OpenMP. We present two solutions to enable OpenMP support (you need to do only
+one).
 
-You first need to install the OpenMP library::
+Using conda
+~~~~~~~~~~~
 
-    brew install libomp
+    One solution is to install another compiler which supports OpenMP. If you
+    use the conda package manager, you can install the ``compilers``
+    meta-package from the conda-forge channel, which provides OpenMP-enabled C
+    compilers.
 
-Then you need to set the following environment variables::
+    It is recommended to use a dedicated conda environment to build
+    scikit-learn from source::
 
-    export CC=/usr/bin/clang
-    export CXX=/usr/bin/clang++
-    export CPPFLAGS="$CPPFLAGS -Xpreprocessor -fopenmp"
-    export CFLAGS="$CFLAGS -I/usr/local/opt/libomp/include"
-    export CXXFLAGS="$CXXFLAGS -I/usr/local/opt/libomp/include"
-    export LDFLAGS="$LDFLAGS -Wl,-rpath,/usr/local/opt/libomp/lib -L/usr/local/opt/libomp/lib -lomp"
+        conda create -n sklearn-dev python numpy scipy cython joblib pytest \
+            conda-forge::compilers conda-forge::llvm-openmp
```

Comment:
We can install pip to avoid messages about a tracker when using `--verbose`:
```suggestion
            pip conda-forge::compilers conda-forge::llvm-openmp
```

Reply:
> messages about a tracker 

Can you explain ?

Reply:
I am not getting the message anymore. It was most likely due to not having internet connection at the time.